src/sentry/lang/javascript/errormapping.py

```diff
@@ -112,15 +113,25 @@ def rewrite_exception(data):
     in place and returns `True` if a modification was performed or `False`
     otherwise.
     """
+    meta = Meta(data.get("_meta"))
     rv = False
-    for exc in get_path(data, "exception", "values", filter=True, default=()):
+
+    values_meta = meta.enter("exception", "values")
+    for index, exc in enumerate(get_path(data, "exception", "values", filter=True, default=())):
         for processor in error_processors.values():
             try:
+                original_value = exc.get("value")
                 if processor.try_process(exc):
+                    values_meta.enter(index, "value").add_remark(
+                        {"rule_id": "react_minified_error", "type": "s"}, original_value
```

Comment:
Not sure about `react_minified_error` rule id, but I assume this is required. @jan-auer, any suggestions?

Reply:
These rule IDs usually refer to data scrubbing rules. All builtin rules are prefixed with `@`, for example `@ip:replace`. 

In our case we're not doing data scrubbing, so what if we create a new namespace for this, for instance: `@processing:react_minified_error` (or a shorter version of this)?

Reply:
I named it `@processing:react`. `react` is grabbed from the processor name itself, so this should be more extensible as well.